packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/InputAwareWebView.java

```diff
@@ -9,7 +9,7 @@
 
 /**
  * A WebView subclass that mirrors the same implementation hacks that the system WebView does in
- * order to correctly create an InputConnection.
+ * Android versions below N in order to correctly create an InputConnection on the IME thread.
```

Comment:
nit: The IME thread is an Android WebView term, maybe we should say "...the IME thread (the WebView's dedicated input thread)".